odoo/tools/mail.py

```diff
@@ -176,7 +176,23 @@ def parse_style(self, el):
                 del el.attrib['style']
 
 
-def html_sanitize(src, silent=True, sanitize_tags=True, sanitize_attributes=False, sanitize_style=False, sanitize_form=True, strip_style=False, strip_classes=False):
+def html_sanitize(src, silent=True, sanitize_tags=True, sanitize_attributes=False, sanitize_style=False,
+                  sanitize_form=True, strip_style=False, strip_classes=False, sanitize_extra_tags=None):
+    """Sanitize HTML content to prevent XSS attacks or unwanted content.
+
+    This function wraps`lxml.html.clean.Cleaner` to process and sanitize HTML content according to the provided arguments.
+
+    :param str src: The HTML source string to be sanitized.
+    :param bool silent: If True, suppresses most of the errors that may occur during sanitization. If False, errors may be raised.
+    :param bool sanitize_tags: If True, sanitizes HTML tags to remove potentially dangerous tags such as <script>, <iframe>, etc.
+    :param bool sanitize_attributes: If True, filters out harmful attributes from tags that can lead to XSS attacks, such as "onclick".
+    :param bool sanitize_style: If True, cleans up CSS styles within HTML elements to prevent malicious code execution.
+    :param bool sanitize_form: If True, removes form tags and kills inputs.
+    :param bool strip_style: If True, strips out all style attributes from the HTML elements.
+    :param bool strip_classes: If True, removes all class attributes from the HTML elements.
+    :param list|tuple sanitize_extra_tags: Additional tags to sanitize beyond those in `tags_to_kill` and `tags_to_remove`.
```

Comment:
because someone once bothered me on a review because I wasn't using the actual documented type but that the implementation actually just required anything iterable
```suggestion
    :param collections.abc.Iterable sanitize_extra_tags: Additional tags to sanitize beyond those in `tags_to_kill` and `tags_to_remove`.
```

Reply:
People would probably use `Iterable` a whole lot more if writing it wasn't so needlessly verbose :eyes: @python-typehint-devs

Reply:
Yes, I can't even import it to only use "Iterable" because it gets flagged as non-used import...

Reply:
you shouldn't have to import anything as it is inside the docstring

Reply:
Only if I import it does my IDE fully recognize it with `Iterable` instead of `collections.abc.Iterable`.

Reply:
then write collection.abc.Iterable, no big deal